python/perftest/templates/_macros.j2

```diff
@@ -54,7 +64,7 @@ args:
   - -c
   - |
       set -e
-      until [ -s /etc/kube-perftest/hosts ]; do
+      until [ -s /etc/kube-perftest/all-hosts ]; do
```

Comment:
This has to be `/etc/kube-perftest/hosts`, because of how the discovery works. Basically, `all-hosts` contains the list of all expected pod names, and is populated _when the benchmark is created_. `hosts` contains the IP addresses of the pods (in `/etc/hosts` format) and is only populated once there is an IP for all the expected pods (read from `all-hosts`).